timescaledb/quick-start/python.md

```diff
@@ -226,73 +221,100 @@ Congratulations, you've successfully inserted data into TimescaleDB using Python
 
 While using `psycopg2` by itself may be sufficient for you to insert rows into your
 hypertable, if you need quicker performance, you can use
-[pgcopy][pg-copy-docs]. To do this, install `pgcopy` using `pip3` or the like and
+[pgcopy][pg-copy-docs]. To do this, install `pgcopy` [using pip][pgcopy-install] and
 then add this line to your list of `import` statements:
 
 ```python
 from pgcopy import CopyManager
 ```
 
-Here's some sample code which shows how to insert data into Timescale using `pgcopy`,
-using the example of sensor data from four sensors:
-
-```python
-#insert using pgcopy
-def fast_insert(conn):
-   cur = conn.cursor()
-
-   #for sensors with ids 1-4
-   for id in range(1,4,1):
-       data = (id, )
-       #create random data
-       simulate_query = """SELECT  generate_series(now() - interval '24 hour', now(), interval '5 minute') AS time,
-       %s as sensor_id,
-       random()*100 AS temperature,
-       random() AS cpu
-       """
-       cur.execute(simulate_query, data)
-       values = cur.fetchall()
-
-       #define columns names of the table you're inserting into
-       cols = ('time', 'sensor_id', 'temperature', 'cpu')
-
-       #create copy manager with the target table and insert!
-       mgr = CopyManager(conn, 'sensor_data', cols)
-       mgr.copy(values)
-
-   #commit after all sensor data is inserted
-   #could also commit after each sensor insert is done
-   conn.commit()
-
-   #check if it worked
-   cur.execute("SELECT * FROM sensor_data LIMIT 5;")
-   print(cur.fetchall())
-   cur.close()
-```
-
-
 ### Step 1: Get data to insert into database
 First we generate random sensor data - you would replace this step with funneling
 in your real data from your data pipeline.
```

Comment:
I know we don't do this elsewhere, but since we don't specifically talk about `generate_series` and how this SQL works, it might be nice to at least say something like:

```suggestion
First we generate random sensor data using the `generate_series` function provided by PostgreSQL.
In the example query below, we will insert a total of 480 rows of data (4 readings, every 5 minutes, for 24 hours).
In your application, this would be the query that saves your time-series data into the hypertable.
in your real data from your data pipeline.
```

Reply:
switch "we" for "you"

Reply:
https://github.com/timescale/docs/commit/eeb197b31ef0439ee0bcda1035199a3d0ccc93e4